qiskit_optimization/applications/graph_optimization_application.py

```diff
@@ -35,14 +38,19 @@ class GraphOptimizationApplication(OptimizationApplication):
     An abstract class for graph optimization applications.
     """
 
-    def __init__(self, graph: Union[nx.Graph, np.ndarray, List]) -> None:
+    def __init__(self, graph: Union[rx.PyGraph, nx.Graph]) -> None:
```

Comment:
Fwiw you can still pass in an adjacency matrix as a numpy array here with the from_adjacency_matrix constructor method: https://qiskit.org/documentation/retworkx/dev/stubs/retworkx.PyGraph.from_adjacency_matrix.html#retworkx.PyGraph.from_adjacency_matrix

Reply:
Thanks. I change it in the latest commit.